uptime-kuma.json

```diff
@@ -131,116 +313,159 @@
-                  "text": "EXPIRED"
+                  "text": "Down"
                 },
                 "1": {
-                  "color": "green",
+                  "color": "dark-green",
+                  "index": 2,
+                  "text": "Up"
+                },
+                "2": {
+                  "color": "dark-yellow",
                   "index": 1,
-                  "text": "VALID"
+                  "text": "Pending"
                 }
               },
               "type": "value"
             }
           ],
+          "noValue": "Everything UP",
           "thresholds": {
             "mode": "absolute",
             "steps": [
-              {
-                "color": "red",
-                "value": null
-              },
               {
                 "color": "green",
-                "value": 1
+                "value": null
               }
             ]
-          }
+          },
+          "unit": "bool"
         },
         "overrides": []
       },
       "gridPos": {
-        "h": 8,
-        "w": 9,
-        "x": 12,
+        "h": 28,
+        "w": 4,
+        "x": 10,
         "y": 0
       },
-      "id": 5,
+      "id": 2,
       "options": {
-        "colorMode": "background",
-        "graphMode": "area",
-        "justifyMode": "center",
-        "orientation": "auto",
+        "colorMode": "background_solid",
+        "graphMode": "none",
+        "justifyMode": "auto",
+        "orientation": "horizontal",
         "reduceOptions": {
           "calcs": [
-            "lastNotNull"
+            "last"
           ],
           "fields": "",
           "values": false
         },
+        "showPercentChange": false,
         "text": {},
-        "textMode": "auto"
+        "textMode": "name",
+        "wideLayout": true
       },
-      "pluginVersion": "8.1.1",
+      "pluginVersion": "10.4.1",
       "targets": [
         {
-          "exemplar": true,
-          "expr": "monitor_cert_is_valid",
-          "interval": "",
-          "legendFormat": "{{ monitor_name }}",
-          "refId": "A"
+          "datasource": {
+            "type": "prometheus",
+            "uid": "SUpEVGqnk"
+          },
+          "disableTextWrap": false,
+          "editorMode": "builder",
+          "exemplar": false,
+          "expr": "uptime_status{instance=\"$Instance\", job=\"$job\", monitor_type!=\"group\"} == 2",
+          "format": "time_series",
+          "fullMetaSearch": false,
+          "hide": false,
+          "includeNullMetadata": true,
+          "instant": true,
+          "legendFormat": "{{monitor_name}}",
+          "range": false,
+          "refId": "B",
+          "useBackend": false
         }
       ],
-      "title": "TLS Certificate Status",
+      "title": "Pending Services",
       "type": "stat"
     },
     {
-      "datasource": null,
+      "datasource": {
+        "type": "prometheus",
+        "uid": "SUpEVGqnk"
+      },
+      "description": "",
```

Comment:
could you add a description? I am lost what this section does

Reply:
```suggestion
      "description": "This panel shows the different http based services and their response times in the selected interval as well as their mean- and minimum response time.",
```